packages/launchpad/src/error/BaseError.cy.tsx

```diff
@@ -1,20 +1,17 @@
 import { codeFrameColumns } from '@babel/code-frame'
 import BaseError from './BaseError.vue'
-import Button from '@cy/components/Button.vue'
 import { BaseErrorFragmentDoc } from '../generated/graphql-test'
 import dedent from 'dedent'
 
 // Selectors
 const headerSelector = '[data-testid=error-header]'
 const messageSelector = '[data-testid=error-message]'
 const retryButtonSelector = '[data-testid=error-retry-button]'
-const customFooterSelector = '[data-testid=custom-error-footer]'
+const docsButtonSelector = '[data-testid=error-docs-button]'
```

Comment:
Not really related to your changes, but we could add some semantics to these selectors (I skipped messageSelector since it would be `[data-testid=error-message] p` which seems weirdly specific to me. But something like this would be good:

```js
const headerSelector = 'h1[data-testid=error-header]'
const messageSelector = '[data-testid=error-message]'
const retryButtonSelector = 'button[data-testid=error-retry-button]'
const docsButtonSelector = 'a[data-testid=error-docs-button]'
```